solutions/agents/variables.tf

```diff
@@ -174,42 +174,42 @@ variable "cloud_monitoring_metrics_filter" {
     type = string
     name = string
   }))
-  description = "To filter custom metrics, specify the Cloud Monitoring metrics to include or to exclude. See https://cloud.ibm.com/docs/monitoring?topic=monitoring-change_kube_agent#change_kube_agent_inc_exc_metrics."
+  description = "To filter on custom metrics, specify the IBM Cloud Monitoring metrics to include or exclude. For more information about customizing the monitoring agent configuration, see [Including and excluding metrics](https://cloud.ibm.com/docs/monitoring?topic=monitoring-change_kube_agent#change_kube_agent_inc_exc_metrics)."
   default     = [] # [{ type = "exclude", name = "metricA.*" }, { type = "include", name = "metricB.*" }]
   validation {
     condition     = length(var.cloud_monitoring_metrics_filter) == 0 || can(regex("^(include|exclude)$", var.cloud_monitoring_metrics_filter[0].type))
-    error_message = "Invalid input for `cloud_monitoring_metrics_filter`. Valid options for 'type' are: `include` and `exclude`. If empty, no metrics are included or excluded."
+    error_message = "The specified `type` for the `cloud_monitoring_metrics_filter` is not valid. Specify either `include` or `exclude`. If the value for `type` is not specified, no metrics are included or excluded."
   }
 }
 
 variable "cloud_monitoring_agent_tags" {
   type        = list(string)
-  description = "List of tags to associate to all matrics that the agent collects. NOTE: Use the 'cloud_monitoring_add_cluster_name' variable to add the cluster name as a tag."
+  description = "A list of the tags to associate with the metrics that the IBM Cloud Monitoring agent collects. To add the cluster name as a tag, use the `cloud_monitoring_add_cluster_name` variable."
   default     = []
   nullable    = false
 }
 
 variable "cloud_monitoring_add_cluster_name" {
   type        = bool
-  description = "If true, configure the cloud monitoring agent to attach a tag containing the cluster name to all metric data."
+  description = "When set to `true`, this variable configures the IBM Cloud Monitoring agent to attach a tag that contains the cluster name to all log messages."
   default     = true
 }
 
 variable "cloud_monitoring_agent_name" {
-  description = "Cloud Monitoring agent name. Used for naming all kubernetes and helm resources on the cluster. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of the IBM Cloud Monitoring agent that is used to name the Kubernetes and Helm resources on the cluster. If a prefix input variable is passed, the name of the IBM Cloud Monitoring agent is prefixed to the value in the `<prefix>-value` format."
```

Comment:
```suggestion
  description = "The name of the IBM Cloud Monitoring agent for all Kubernetes and Helm resources in the cluster. If a value is specified in the `prefix` input, the name is set as `<prefix>-name`."
```

Reply:
@maheshwarishikha can you provide more details about where the prefix is coming from?